pr_check.sh

```diff
@@ -29,8 +29,7 @@ source $CICD_ROOT/build.sh
 source $CICD_ROOT/deploy_ephemeral_env.sh
 
 # Run smoke tests using a ClowdJobInvocation and iqe-tests
-COMPONENT_NAME="provisioning"  # name of app used in deploy-iqe-cji
```

Comment:
So should this be `provisioning-api`/`provisioning-backend`? what will `COMPONENT_NAME` be if we do not set it here?

Reply:
It's set at the top of the file to `provisioning-backend`, and should match whatever the component/clowdapp name is. This is what's used to do endpoint lookups in clowder envs for CJI pipelines.